ui_framework/doc_site/src/views/card/card_group.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import {
+  KuiCardGroupContainer,
+  KuiCardGroup,
+  KuiCard,
+  KuiCardDescription,
+  KuiCardDescriptionTitle,
+  KuiCardDescriptionText,
+  KuiCardFooter
+} from '../../../../components';
+
+export default () => {
+  const cardStyle = {
+    width: '400px'
+  };
+
+  return (
+    <div>
+      <KuiCardGroupContainer>
+        <KuiCardGroup>
+          <KuiCard className="kuiCardGroup__card" style={cardStyle}>
```

Comment:
I think the `kuiCardGroup__card` class should be eliminated and its content should go into a nested CSS declaration like
```
.kuiCardGroup > .kuiCard {
    ...
}
```
to make the life of the user easier.
It's probably not completely BEM but sounds logical for me.

And there are two different `.kuiCardGroup__card` definitions in:
1. ui_framework/components/card/_card_group.scss
2. ui_framework/dist/ui_framework.scss (and only this has effect)

One should be enough, shouldn't it? :)